coralproject/talk
# I18n

## What does this PR do?

Updates internationalization across talk.

## How do I test this PR?

1. Load the embed stream.
2. Enter `localStorage.setItem('locale','es-US')` in the browser console.
3. Text should switch to spanish (some translations missing.)

## Notes

- After discussions with @dan and @gaba I have decided to go with a consistent translation file for each package. All languages are included in the same file to avoid the need to modify imports when adding languages.
- Some spanish translations have been marked as todo with ""¡traduceme!"" rather than trusting google translate.

@coralproject/tech
